batch/setup.py

```diff
@@ -6,19 +6,6 @@
     url = 'https://github.com/hail-is/hail.git',
     author = 'Hail Team',
     author_email = 'hail@broadinstitute.org',
-    description = 'Kubernetes job manager client',
-    packages = find_packages(),
-    include_package_data=True,
-    install_requires=[
-        'cerberus',
-        'kubernetes',
-        'flask',
```

Comment:
Where did these requirements go?

Reply:
So it seems like dependencies are duplicated: here and the requirements.txt for base.  I decided to start specifying just the dependencies in base.  Many packages are used in different projects, so it seemed like there was less duplication if base requirements.txt was the reference.

It's not just the packages, we're pinning packages now, so versions will be duplicated (and therefore probably out of sync), see: https://github.com/hail-is/hail/commit/fab8d834b5b252013d8c077915ea4e2a9cd33a71